### ISSUE TITLE
Openmaptiles process: argument mismatch:userdata,string,nil ()

### ISSUE THREAD

**Issue by sasfeat** [systemed/tilemaker#153]
Got error making tiles from [russia-northwestern-fed-district](http://download.geofabrik.de/russia/northwestern-fed-district.html). However everything works perfectly with [liechtenstein](http://download.geofabrik.de/europe/liechtenstein.html).

```
$ docker run -v /data/tiles:/data -it tilemaker --input /data/northwestern-fed-district-latest.osm.pbf --output=/data/nw.mbtiles --config /resources/config-openmaptiles.json --process /resources/process-openmaptiles.lua --verbose
Layer aerodrome_label (z10-14)
Layer place (z4-14)
Layer place_detail (z14-14) -> place
Layer poi (z12-14)
Layer poi_detail (z14-14) -> poi
Layer housenumber (z14-14)
Layer waterway (z9-14)
Layer waterway_detail (z12-14) -> waterway
Layer transportation (z8-14)
Layer transportation_main (z9-14) -> transportation
Layer transportation_mid (z11-14) -> transportation
Layer transportation_detail (z13-14) -> transportation
Layer transportation_name (z8-14)
Layer transportation_name_mid (z12-14) -> transportation_name
Layer transportation_name_detail (z14-14) -> transportation_name
Layer building (z13-14)
Layer water (z9-14)
Layer water_name (z9-14)
Layer water_name_detail (z12-14) -> water_name
Layer aeroway (z11-14)
Layer park (z11-14)
Layer landuse (z11-14)
Layer landcover (z11-14)
Layer mountain_peak (z11-14)
Reading /data/northwestern-fed-district-latest.osm.pbf
lua runtime error: maybe...Argument mismatch:userdata,string,nil         candidate is:
                OsmLuaProcessing,std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >,std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >,

terminate called after throwing an instance of 'kaguya::LuaTypeMismatch'
  what():  type mismatch!!
```

**Comment by systemed** [comment]
This is likely to be because a tag somewhere in the extract is tripping up the OpenMapTiles-compatible Lua script. Probably it's getting a string where a number is expected, or nil, or something like that. If I get a minute I'll look into it, but if you want to hunt through to debug it yourself, that's where I'd start.

**Comment by sasfeat** [comment]
Fixed #154
